polynote-frontend/polynote/data/messages.ts

```diff
@@ -707,6 +754,8 @@ Message.codecs = [
     RunningKernels,  // 24
     RenameNotebook,  // 25
     DeleteNotebook,  // 26
+    null as unknown as typeof Message, // 27 - reserve for CopyNotebook
```

Comment:
Left this space for #763 (one of us will have to merge, but at least it will be easy)